chips/cc26x2/src/prcm.rs

```diff
@@ -134,6 +134,16 @@ register_bitfields![
         // RESERVED (bits 1-31)
         CLK_EN      OFFSET(0) NUMBITS(1) []
     ],
+    // Clock gate type for when there are two peripherals
+    ClockGate2 [
+        // RESERVED (bits 1-31)
```

Comment:
I don't understand this comment, bit 0 and bit 1 is `CLK_EN`?

Reply:
I refactored this section as I realized after more bitfields experience that my usage was not canonical

Reply:
But to answer you question, each peripheral has it's own bit. So bit 0 control clocks towards UART0 and bit 1 control clocks towards UART1.